mesonbuild/modules/gnome.py

```diff
@@ -222,6 +222,27 @@ class MkEnums(_MkEnumsCommon):
     KwargInfo('symbol_prefix', (str, NoneType)),
 ]
```

Comment:
Dang github won't let me put the comment in the right place, but on line 127, you need to replace `annotations: List[str]` with `annotations: List[List[str]]`

Reply:
Oops, fixed.